compiler/p/codegen/FPTreeEvaluator.cpp

```diff
@@ -562,11 +562,10 @@ TR::Register *OMR::Power::TreeEvaluator::dloadEvaluator(TR::Node *node, TR::Code
    return TR::TreeEvaluator::dloadHelper(node, cg, tempReg, TR::InstOpCode::lfd);
    }
 
-
-
 TR::Register *OMR::Power::TreeEvaluator::vsplatsEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
    TR::Node *child = node->getFirstChild();
+   static bool disableDirectMove = feGetEnv("TR_disableDirectMove") ? true : false;
```

Comment:
A environment variable was used there to match the one used inside generateMvFprGprInstructions() in tr.open/omr/compiler/p/codegen/GenerateInstructions.cpp. If TR_disableDirectMove is set, my code changes do not want to call generateMvFprGprInstructions anymore. The reason for this is if TR_disableDirectMove is set, a code path that's meant for float registers rather than vector registers in general will be used. Maybe a better solution would be to change generateMvFprGprInstructions to properly support all vector registers?